### PR TITLE
Check transfers return values

### PR DESCRIPTION
**Pull request by facuspagnuolo**
Fixes https://github.com/zeppelinos/kernel/issues/28

### CODE REVIEW

File: examples/complex/contracts/DonationsV1.sol [hunk 1]
```diff
@@ -21,8 +21,7 @@ contract DonationsV1 is Ownable {
   }
 
   function withdraw(address _wallet) onlyOwner {
-
     // Withdraw all donated funds.
-    _wallet.transfer(this.balance);
+    require(_wallet.transfer(this.balance));
```
Comment (frangio) [inline examples/complex/contracts/DonationsV1.sol]: This is not the kind of `transfer` whose return value we need to check. Only token transfers.

File: examples/complex/contracts/DonationsV2.sol [hunk 2]
```diff
@@ -25,7 +25,7 @@ contract DonationsV2 is DonationsV1 {
     if(msg.value > highestDonation) {
 
       // Emit a token.
-      token.mint(msg.sender, numEmittedTokens);
+      require(token.mint(msg.sender, numEmittedTokens));
```
Comment (frangio) [inline examples/complex/contracts/DonationsV2.sol]: `mint` also doesn't have a return value. :sweat_smile:

**Comment by frangio** [comment]
I didn't find any unchecked boolean return values in this repo. (In fact practically no boolean return values.)

**Comment by facuspagnuolo** [comment]
Totally agree, I don't know what I was thinking when I did this 😅

**Pull request closed without merging**